docs/UserGuide.md

```diff
@@ -289,6 +347,42 @@ If your changes to the data file makes its format invalid, FApro will discard al
 Furthermore, certain edits can cause FApro to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 </box>
 
+### (_Coming Soon_) Uploading a profile photo for a client : `image`
+
+Adds a profile photo for a specified client in FApro.
+
+Format: `image INDEX i/PATH`
+
+* Adds the profile photo for the client at the specified `INDEX` with the image found at `PATH`.
+* The index refers to the index number shown in the displayed client list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* The path refers to the relative path of the image to the main repository.
+* If there is no photo uploaded, the profile photo section will display a placeholder profile photo.
+
+Examples:
+* `list` followed by `image 3 i/profiles/charlotte.png` adds a profile photo found at `profiles/charlotte.png` to the 3rd client in FApro.
+* `find david` followed by `image 1 i/profiles/david.png` adds a profile photo found at `profiles/david.png` to the 1st client in the results of the `find` command.
+
+### (_Coming Soon_) Adding notes for a client : `addnote`
+
+Adds note for a specified client in FApro. This feature can be used
+* during preparation for appointment: to take note of what to say during it
+* during the appointmeent: to take note of what the client is saying
+* after the appointment: to take not of the follow-up actions
+
+Format: `addnotes INDEX note/NOTE [note/NOTE]`
```

Comment:
'addnote' instead

Reply:
I don't really understand the format. Should it be `addnote INDEX note/[NOTE]...`?

Reply:
Oh, this one means like, must have at least one note. 
... can be used multiple times including zero times, so I added note/NOTES infront